db/models.py

```diff
@@ -1 +1,39 @@
 from django.db import models
+from django.db.models import CharField
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+    def __str__(self) -> CharField:
+        return self.name
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+    def __str__(self) -> CharField:
```

Comment:
Although it could raise warnings in different IDEs, it would be more preferable to indicate that the return type is `str` for the `__str__` method, because it would be a value that is eventually returned, not an instance of the `CharField`. Wrapping the `self.name` in f-string could be a workaround for this issue, but it leads to an unnecessary line formatting that can be omitted. For this specific case, ignore warnings in the IDE, indicate that the return type is `str`, and leave the return as it is with a single value. It relates only to cases when you return a single value for a single `CharField` when you combine different fields in the `return` using f-string.